iothub/service/src/ServiceClient.cs

```diff
@@ -66,25 +67,25 @@ public void Dispose()
         protected virtual void Dispose(bool disposing) { }
 
         /// <summary>
-        /// Create ServiceClient from the specified connection string using specified Transport Type
+        /// Create an instance of ServiceClient from the specified IoT Hub connection string using specified Transport Type.
```

Comment:
Adding `see<cref="TransportType ">` will be helpful here and below in the transportType param